### ISSUE TITLE
Interest in translating Comprehensive Rust 🦀?

### ISSUE THREAD

**Issue by mgeisler** [tomoyuki-nakabayashi/book#41]
Hi there,

I found this translated book and I was wondering if people here would be interested in helping translate [Comprehensive Rust 🦀: Bare Metal](https://google.github.io/comprehensive-rust/bare-metal.html)?

**Comment by tomoyuki-nakabayashi** [comment]
Hi,

Yes, I'm (and may be some Japanese) interested in translating Comprehensive Rust.
Can I participate in the translation by following this instruction and creating a pull request?
https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md

**Comment by mgeisler** [comment]
> Can I participate in the translation by following this instruction and creating a pull request?
> https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md

Yes, that would be fantastic! Feel free to make a first PR with an empty `ja.po` file just to get the infrastructure setup.
